chapter_appendix-tools-for-deep-learning/utils.md

```diff
@@ -261,6 +261,13 @@ def evaluate_accuracy_gpu(net, data_iter, device=None): #@save
     return metric[0] / metric[1]
 
 
+#@save
+def init_cnn_weights(m):
```

Comment:
Let's change this to `layer`. `m` was chosen earlier because of how [pytorch documentation](https://pytorch.org/docs/stable/generated/torch.nn.Module.html#torch.nn.Module.apply) shows it as a submodule.